app.rb

```diff
@@ -193,6 +222,8 @@ def scopes_for(ial)
         'openid email x509'
       when '2', 'biometric-comparison-required'
         'openid email profile social_security_number phone address x509'
+      when '2', 'enhanced-ipp-required'
```

Comment:
This duplicate when clause is going to be ignored. Is the ial '2' for enhanced-ipp?

Reply:
Done. Resolved with commit c23a68dc35af6686a7f18a66e8062544a24f5ec0